### ISSUE TITLE
Enhancement - Ability to manage /etc/raddb/clients.conf

### ISSUE THREAD

**Issue by lmorfitt** [raphink/puppet-freeradius#3]
Hello,

I would like to have the ability to manage the clients.conf using a defined resource type, which maintains the RHEL standards as best as possible. 

Something similar to https://github.com/djjudas21/puppet-freeradius/blob/master/manifests/client.pp without a separate raddb/clients.d/ folder. 

This allows to keep compatibility with the out the box config as best as possible.

thanks
Luke

**Comment by raphink** [comment]
I agree this would be useful. I built this module entirely in a Test Driven way, which means I only added what I needed, when I needed it, after adding tests for the missing feature. For now, `clients.conf` hasn't been a requirement in my case, so it's not implemented yet.

I'd encourage you to provide tests (at least acceptance tests, if possible unit tests, too) for this feature (and even the Puppet code if you're willing!).
